cmd/main.go

```diff
@@ -226,3 +238,32 @@ func createS3Bucket(objectHandler s3wrapper.API) {
 		}
 	}
 }
+
+func NewApiEnabler(h http.Handler, log logrus.FieldLogger) *ApiEnabler {
+	return &ApiEnabler{
+		log:       log,
+		isEnabled: false,
+		inner:     h,
+	}
+}
+
+type ApiEnabler struct {
+	log       logrus.FieldLogger
+	isEnabled bool
+	inner     http.Handler
+}
+
+func (a *ApiEnabler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if !a.isEnabled {
```

Comment:
Don't we need the `/ready` endpoint to always return true, even it the isEnabled is false?

Reply:
```
health api will indicate the service is started and have all the needed clients
ready will indicate that service is ready to receive API calls and will start to be active only
after dummy iso is generated
Using middleware to enable the api after the dummy iso is generated
```
i was thinking that it will be clearer if `healthy` will indicate that we started to run and will not crash again and `ready` indicate that we can receive APIs from the users

Reply:
ok